Readme.md

```diff
@@ -199,6 +210,30 @@ This event is triggered if `console.log` is used on the page. But this event is
 ##### .on('page-alert', message)
 This event is triggered if `alert` is used on the page.
 
+##### .on('download', event, downloadItem)
+This event is triggered when Electron emits `'will-download'`.  This event is also emitted after downloads are started when [`DownloadItem`](https://github.com/atom/electron/blob/master/docs/api/download-item.md) emits `'updated'` or `'done'`.   The possible values for `event` are `'started'`, `'cancelled'`, `'interrupted'`, or `'completed'`.  Note that by listening to `'download'`, Nightmare expects the default download behavior to be overridden. 
+
+Downloads should be completed using `Nightmare.emit('download', action, downloadItem)`.  The possible values for `action` are `'cancel'`, `'continue'` for default behavior, or a file path (file name and extension inclusive) to save the download to an alternative location. The `downloadItem` parameter should use the item passed by `'download'`.
+
+For example:
+
+```javascript
+var nightmare = Nightmare();
+nightmare.on('download', function(event, downloadItem){
+  if(state == 'start'){
```

Comment:
Ah, man, I missed the rename.  It should be `state == 'started'`, with `event` renamed to `state`.  The event name is `download`, and I wanted to be clear about what the first parameter was.

Also, `'start'` should be `'started'`.  Fix coming.  Thanks for the find!